pkg/controller/controller_utils.go

```diff
@@ -213,6 +214,8 @@ func NewControllerExpectations() *ControllerExpectations {
 type PodControlInterface interface {
 	// CreateReplica creates new replicated pods according to the spec.
 	CreateReplica(namespace string, controller *api.ReplicationController) error
+	// CreateReplicaOnNodes creates a new pod according to the spec, on a specified list of nodes.
+	CreateReplicaOnNode(namespace string, controller *expapi.Daemon, nodeNames string) error
```

Comment:
Sorry it should be "CreateReplicaOnNode(namespace string, controller *expapi.Daemon, nodeName string) error"
It creates a replica on a single node, it appears I added an 's' to 'nodeNames' by mistake.